plugins/modules/network_settings_intent.py

```diff
@@ -140,7 +156,7 @@
           ipv6_subnet:
             description: IPv6 Subnet address, example 2001 db8 85a3 0 100.
             type: str
-          ipv6TotalHost:
+          ipv6_total_host:
             description: IPv6 total host is required when ipv6_prefix value is false.
             type: int
           name:
```

Comment:
description: Name of the reserved IP subpool.

prev_name:
description: The former name associated with the reserved IP sub-pool

site_name:
description: The name of the site provided as a path parameter, used to specify where the IP sub-pool will be reserved.